File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/43_compito_java_20230221/Terapista.py
from Operatore import Operatore


class Terapista(Operatore):
    def __init__(self, codice, tipo, nome, costo_visita, terapia, studio, n_sedute_terapia):
        super().__init__(codice, tipo, nome, costo_visita)
        self._terapia = terapia
        self._studio = studio
        self._n_sedute_terapia = n_sedute_terapia

    def __str__(self):
        return super().__str__() + "\t-\t-\t-\t" + self._terapia + "\t" + self._studio + "\t" + str(self._n_sedute_terapia) + "\tterapista"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/43_compito_java_20230221/Terapista.java
public class Terapista extends Operatore {
    private String terapia, studio;
    private int n_sedute_terapia;

    public Terapista(int codice, String tipo, String nome_operatore, double costo_visita, String terapia, String studio,
            int n_sedute_terapia) {
        super(codice, tipo, nome_operatore, costo_visita);
        this.terapia = terapia;
        this.studio = studio;
        this.n_sedute_terapia = n_sedute_terapia;
    }

    public String toString() {
        return super.toString() + "\t-\t-\t-\t" + terapia + "\t" + studio + "\t" + n_sedute_terapia + "\tterapista";
    }
}